python/housinginsights/sources/wmata_distcalc.py

```diff
@@ -133,10 +134,11 @@ def get_dist(self, unique_data_ids=None, sample=False, output_type='csv',db='loc
                 self._array_to_csv(self.distHeader, self.distOutput, self.output_paths[u])
```

Comment:
Unrelated to changes in this PR but caught my eye, maybe in future we can reuse result_to_csv in parent base class. https://github.com/codefordc/housing-insights/blob/dev/python/housinginsights/sources/base.py#L128